.github/actions/ci-setup/action.yml

```diff
@@ -6,6 +6,39 @@ runs:
   using: composite
 
   steps:
+    - uses: cargo-bins/cargo-binstall@main
+    - name: Make more disk space available on public runner
+      run: |
```

Comment:
@jjeangal Not urgent, just a thought.

 We should see if this is something we will still need to do on the buildjet runners,
If we don't need to do this, then I think we can create a way to optionally enable this in the CI for runners that need it.

Reply:
We also want to limit the workflows that really need more disk space as these have costs. But if you think it's needed we can enable buildjet.